core/src/main/resources/filodb-defaults.conf

```diff
@@ -25,11 +25,11 @@ filodb {
 
     stale-sample-after = 5 minutes
 
-    # Maximum number of RangeVectors returned from any sub-plan QueryResult
-    # vectors-limit = 150
+    # Maximum number of samples to return in a query
+    sample-limit = 1000000
 
-    # Maximum number of samples to return per RangeVector/result time series
-    sample-limit = 200
+    # Minimum step required for a query
+    min-step = 10 seconds
```

Comment:
5s default would be good. I have seen 5s scrape interval in monstor host prometheus configs